web/src/ProviderEditPage.js

```diff
@@ -207,7 +207,7 @@ class ProviderEditPage extends React.Component {
         return Setting.getLabel(i18next.t("provider:App key"), i18next.t("provider:App key - Tooltip"));
```

Comment:
You forgot to add it in backend Go code

Reply:
> You forgot to add it in backend Go code

What is this parameter for? This platform does not have an option for an app key.

Reply:
@cutedrag0n I said you didn't modify the Go code for your new provider, which is wrong. Did it work if you don't modify it? I doubt so.